docs/schema/codemethodaction-element-in-actions-solutionmanifestdeclarativeextensions-schema.md

```diff
@@ -44,7 +45,7 @@ The following sections describe attributes, child elements, and parent elements.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p>**MethodType**</p></td>
+<td align="left"><p><b>MethodType</b></p></td>
```

Comment:
Regression... don't go from markdown > HTML. 

When submitting changes to an HTML table, go ahead and convert to markdown.